core/chains/evm/txmgr/nonce_syncer.go

```diff
@@ -91,6 +91,9 @@ func (s NonceSyncer) SyncAll(ctx context.Context, keyStates []ethkey.State) (mer
 
 	wg.Add(len(keyStates))
 	for _, keyState := range keyStates {
+		if keyState.Disabled {
+			continue
+		}
```

Comment:
Otherwise we error inside of `fastForwardNonceIfNecessary` because the SQL query is limited to enabled only:
```go
func (orm ksORM) getNextNonce(address common.Address, chainID *big.Int, qopts ...pg.QOpt) (nonce int64, err error) {
	q := orm.q.WithOpts(qopts...)
	err = q.Get(&nonce, "SELECT next_nonce FROM evm_key_states WHERE address = $1 AND evm_chain_id = $2 AND disabled = false", address, chainID.String())
```